packages/python-plum/PKGBUILD

```
@@ -0,0 +1,30 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+# This file is based on the AUR package by Caleb Maclennan <caleb@alerque.com>
+
+_pyname=plum
+pkgname=python-${_pyname,,}
```

```suggestion
pkgname=python-plum
_pkgname=plum-py
```

https://pypi.org/project/plum-py/